libs/boost/Makefile

```diff
@@ -12,18 +12,17 @@
 
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/nls.mk
 include $(INCLUDE_DIR)/target.mk 
 
 PKG_NAME:=boost
-PKG_VERSION:=1_61_0
+PKG_VERSION:=1.62.0
```

Comment:
You can have both PKG_VERSION:=1.62.0 and PKG_SOURCE_VERSION=1_62_0 and use them where appropriate. No need to repeat the 1_62_0 literal for every reference. Makes updating much easier.